### PR TITLE
feat: use flavor for host, scheme and logo

### LINKED ISSUE #193: Flavor issues tracker
- [x] use distinct url scheme for Titan flavors
 - [x] use three environment variables for Titan flavors (ex: `DEV_HOST`, `ALPHA_HOST`, `PROD_HOST`)
 - [x] use images associated with Titan flavor

### PR DESCRIPTION
**Pull request by armanddidierjean**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -10,7 +10,9 @@
             "type": "dart",
             "args": [
                 "--flavor",
-                "dev"
+                "dev",
+				"--dart-define",
```
Comment (julien4215) [inline .vscode/launch.json:14]: We don't need to use `--dart-define` (see the answer to the stackoverflow https://stackoverflow.com/questions/74751159/how-to-read-flavor-parameter-value-from-flutter-code)

**Review by julien4215 — DISMISSED** [review]

**Review by guillaumetavernier — APPROVED** [review]

File: .vscode/launch.json [hunk 2]
```diff
@@ -1,28 +1,37 @@
...
+            "name": "Titan (profile mode)",
+            "request": "launch",
+            "type": "dart",
+            "flutterMode": "profile",
+            "args": [
+                "--flavor",
+                "dev"
+            ]
+        },
+        {
```
Comment (guillaumetavernier) [inline .vscode/launch.json:26]: Maybe we could add the possibility to quickly build an alpha

File: .env.template [hunk 3]
```diff
@@ -1,6 +1,7 @@
 THE_MOVIE_DB_API = "<<Your MovieDB API Token>>"
 # A trailing slash is required
-DEBUG_HOST = "<<Your Debbuging Server Host>>"
-RELEASE_HOST = "<<Your Production Server Host>>"
+DEV_HOST = "<<Your Debbuging Server Host>>"
+ALPHA_HOST = "<<Your Alpha Server Host>>"
```
Comment (guillaumetavernier) [inline .env.template:4]: Right now echo `RELEASE_HOST=$DEBUG_HOST` is used in workflows when building an alpha. 
Plus I think that workflows are not updated to the new `.env` file

**Review by guillaumetavernier — CHANGES_REQUESTED** [review]

File: .env.template (hunk 3 above)
Comment (julien4215) [reply .env.template:4]: Workflows are updated in this PR. We will juste need to rename secrets.

**Review by guillaumetavernier — APPROVED** [review]

**Comment by Petitoto** [comment]
On Android, redirect scheme does not work after login. This should be updated to match the flavor:
https://github.com/aeecleclair/Titan/blob/cc115630a26e6a84ed338b773891621ed3cabb21/android/app/build.gradle#L61-L63

**Comment by julien4215** [comment]
> On Android, redirect scheme does not work after login. This should be updated to match the flavor:
> 
> https://github.com/aeecleclair/Titan/blob/cc115630a26e6a84ed338b773891621ed3cabb21/android/app/build.gradle#L61-L63

It should now be fixed

**Review by Petitoto — APPROVED** [review]

**Comment by Petitoto** [comment]
This would need to be tested on iOS before merging @armanddidierjean

**Pull request merged**